src/nl/surf/eduhub_rio_mapper/specs/common.clj

```diff
@@ -121,6 +122,20 @@
 (s/def ::LongLanguageTypedStrings
   (s/coll-of ::LongLanguageTypedString))
 
+(s/def ::VeryLongLanguageTypedString/language
+  (re-spec #"^[a-z]{2,4}(-[A-Z][a-z]{3})?(-([A-Z]{2}|[0-9]{3}))?$"))
+
+(s/def ::VeryLongLanguageTypedString/value
```

Comment:
Geen idee of 10000 chars genoeg is.... Ik denk dat voor dit veld we beter een `UnlimitedLanguageTypedString/value` kunnen gebruiken, wat denk jij?

Reply:
Ok, doe ik dat.